apps/web/pages/api/trpc/[trpc].ts

```diff
@@ -30,34 +30,64 @@ export default trpcNext.createNextApiHandler({
    * @link https://trpc.io/docs/caching#api-response-caching
    */
   responseMeta({ ctx, paths, type, errors }) {
-    // assuming we have all our public routes in `viewer.public`
-    const allPublic = paths && paths.every((path) => path.startsWith("viewer.public."));
-    // checking that no procedures errored
-    const allOk = errors.length === 0;
-    // checking we're doing a query request
-    const isQuery = type === "query";
-
-    // i18n response depends on request header
-    const nonCacheableQueries = ["viewer.public.i18n"];
-    const isThereANonCacheableQuery = paths?.some((path) => nonCacheableQueries.includes(path));
-    const isThereACacheableQuery = paths?.some((path) => !nonCacheableQueries.includes(path));
-    if (isThereANonCacheableQuery && isThereACacheableQuery) {
-      console.warn(
-        "Cacheable and Non-cacheable queries are mixed in the same request. Not going to cache the request"
-      );
-    }
+    // Some helpers relevant to this function only
+    const ONE_DAY_IN_SECONDS = 60 * 60 * 24;
+    const TWO_HOURS_IN_SECONDS = 60 * 60 * 2;
+
+    // Our response to indicate no caching
+    const noCacheResponse = {};
 
+    // TODO: Confirm this is doing what the original code intended
```

Comment:
Looking to get confirmation that the intent of this line from the original if statement was to not cache anything that's SSR? I'll remove this comment once I have clarity

- [ ] @zomars or @PeerRich to confirm caching is not wanted on SSR